tests/functional/tests.py

```diff
@@ -447,25 +940,21 @@ def main():
     is_s3 = server_endpoint.startswith("s3.amazonaws")
     client = Minio(server_endpoint, access_key, secret_key, secure=secure)
     # Check if we are running in the mint environment.
-    is_mint_env = os.path.exists(os.getenv('DATA_DIR', '/mint/data'))
-
-    _http = urllib3.PoolManager(
-        cert_reqs='CERT_REQUIRED',
-        ca_certs=certifi.where()
-    )
-
-    # Get unique bucket_name, object_name.
-    bucket_name = uuid.uuid4().__str__()
-    object_name = uuid.uuid4().__str__()
+    data_dir = os.getenv('DATA_DIR')
+    if data_dir == None:
+       os.environ['DATA_DIR'] = data_dir = '/mint/data'
+    is_mint_env = (os.path.exists(data_dir) and
+                  os.path.join(data_dir, 'datafile-1-MB') and
+                  os.path.join(data_dir, 'datafile-11-MB'))
 
     # Enable trace
     # import sys
     # client.trace_on(sys.stderr)
 
     testfile = 'datafile-1-MB'
     largefile = 'datafile-11-MB'
-    if is_mint_env:
-        data_dir = os.getenv('DATA_DIR')
+    is_mint_env = False
```

Comment:
Are you sure this line isn't a typo? is_mint_env is just calculated above.

Reply:
This line was for testing purposes and it was supposed to be deleted before push.